.github/workflows/job_keras3_backend.yml

```diff
@@ -0,0 +1,114 @@
+        required: true
+
+permissions: read-all
+
+jobs:
+  Keras3_OpenVINO_Backend:
+    name: Keras 3 OpenVINO Backend
+    timeout-minutes: 10
+    defaults:
+      run:
+        shell: ${{ contains(inputs.runner, 'win') && 'pwsh' || 'bash' }}
+    runs-on: ${{ inputs.runner }}
+    container: ${{ fromJSON(inputs.container) }}
+    env:
+      INSTALL_DIR: ${{ github.workspace }}/install
+      INSTALL_WHEELS_DIR: ${{ github.workspace }}/install/wheels
+      KERAS_REPO: ${{ github.workspace }}/keras_repo
+      KERAS_HOME: ${{ github.workspace }}/keras_repo/.github/workflows/config/openvino
+
+    steps:
+      # Needed as ${{ github.workspace }} is not working correctly when using Docker
+      - name: Setup Variables
+        run: |
+          echo "INSTALL_DIR=$GITHUB_WORKSPACE/install" >> "$GITHUB_ENV"
+          echo "KERAS_REPO=$GITHUB_WORKSPACE/keras" >> "$GITHUB_ENV"
+          echo "KERAS_HOME=$GITHUB_WORKSPACE/keras/.github/workflows/config/openvino" >> "$GITHUB_ENV"
+
+      - name: checkout actions
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        timeout-minutes: 15
+        with:
+          sparse-checkout: |
+            .github/actions/setup_python
+            .github/actions/install_ov_wheels/action.yml
+
+      - name: Clone Keras 3 repository
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        timeout-minutes: 15
+        with:
+          repository: 'keras-team/keras'
+          path: ${{ env.KERAS_REPO }}
+          ref: 'master'
+
+      - name: Download OpenVINO artifacts (wheels)
+        uses: actions/download-artifact@fa0a91b85d4f404e444e00e005971372dc801d16 # v4.1.8
+        with:
+          pattern: openvino_[wheels|openvino_tokenizers]*
+          path: ${{ env.INSTALL_WHEELS_DIR }}
+          merge-multiple: true
+
+      - name: Setup Python ${{ inputs.python-version }}
+        uses: ./.github/actions/setup_python
+        with:
+          version: ${{ inputs.python-version }}
+          pip-cache-path: ${{ runner.os == 'Linux' && env.PIP_CACHE_PATH_LINUX || env.PIP_CACHE_PATH_WIN }}
+          should-setup-pip-paths: ${{ runner.os != 'macOS' }}
+          self-hosted-runner: ${{ runner.os != 'macOS' }}
+
+      - name: Install OpenVINO Python wheels
+        uses: ./openvino/.github/actions/install_ov_wheels
+        with:
+          wheels-dir-path: ${{ env.INSTALL_WHEELS_DIR }}
+          wheels-to-install: 'openvino openvino_tokenizers'
+
+      - name: Install dependencies
+        working-directory: ${{ env.KERAS_REPO }}
+        run: |
+          REQUIREMENTS_FILE="requirements-common.txt"
+          pip install -r $REQUIREMENTS_FILE --upgrade
+          pip uninstall -y keras keras-nightly
+
+      - name: Test integrations
+        working-directory: ${{ env.KERAS_REPO }}
+        run: |
+          python integration_tests/import_test.py --junitxml=${{ env.INSTALL_TEST_DIR }}/TEST-keras3_integration1.xml
```

Comment:
There is no `INSTALL_TEST_DIR` in the `env`.

Reply:
added